qrust-api/src/main/resources/application.yml

```
@@ -2,7 +2,22 @@ spring:
+    hibernate:
+      ddl-auto: create
 
 crypto:
   aes:
     key: ${CRYPTO_AES_KEY}  # Base64 인코딩된 32바이트 키 (AES-256)
+
+minio:
+  endpoint: http://mayfifth99.store:9000
+  access-key: minioadmin
+  secret-key: minioadmin
+  bucketName: qrust-bucket
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)